Lecture8/src/main/java/Shop.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import pageObject.*;
+
+public class Shop {
+    public static String addRandomTractorToCart(String homepageUrl, WebDriver driver) throws InterruptedException {
+        driver.get(homepageUrl);
+        Thread.sleep(5000);
```

Comment:
change to wait

Reply:
implementation was changed to driver setting:
driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);